bf_switchd/bf_switchd.c

```diff
@@ -1687,6 +1687,12 @@ int bf_switchd_lib_init_local(void *ctx_local) {
   }
   #endif
 
+  ret = system("rm /tmp/iospec.io");
```

Comment:
Better to define a macro for IOSPEC file name and use the macro everywhere.
It provides more readability.

Reply:
taken care